### PR TITLE
product page changes

### PR DESCRIPTION
**Pull request by boazmatz**
change buttons to buttons like home page
remove rating button
add mention to creator

**mobile view**
center buttons
center text

### CODE REVIEW

File: public/app.js [hunk 1]
```diff
@@ -3,8 +3,7 @@ const shareData = {
     text: 'Learn web development on MDN!',
     url: 'https://developer.mozilla.org',
 };
+    function tweetCurrentPage()
```
Comment (roeeyud) [inline public/app.js:6]: Does this work? It doesn't make sense.
Also notice the code is a bit messy, format it so it's readable

File: templates/product.html [hunk 2]
```diff
@@ -31,18 +31,18 @@
...
   <div class="row">
-    <div class="col px-md-5">
+    <div class="col px-md-5 text-center">
         <a :href="'' + product.fileslink" download>
-        <button type="button" class="btn btn-success" >download</button>
+        <button type="button" class="btn btn-outline-primary" >download</button>
         </a>
-        <button id="share-button" type="button" class="btn btn-success">share</button>
-        <button type="button" class="btn btn-success">rating</button>
+        <button id="share-button" type="button" class="tweet btn btn-outline-primary" target="_blank">share</button>
```
Comment (roeeyud) [inline templates/product.html:45]: I don't want the word "tweet" in a generic share button

File: public/app.js (hunk 1 above)
Comment (boazmatz) [reply public/app.js:6]: no. we are trying a lot of things

File: public/app.js [hunk 3]
```diff
@@ -2,9 +2,4 @@ const shareData = {
     title: 'MDN',
```
Comment (roeeyud) [inline public/app.js:2]: Can you also delete this code? it isn't used

Comment (boazmatz) [reply public/app.js:2]: but we will write share model

**Pull request merged**